netbox_onboarding/tests/test_views_29.py

```diff
@@ -0,0 +1,56 @@
+"""Unit tests for netbox_onboarding views.
+
+(c) 2020 Network To Code
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from dcim.models import Site
+from utilities.testing import ViewTestCases
```

Comment:
Not to expand the scope of this PR, but out of curiosity did you look into whether we can/should be using `ViewTestCases` for the 2.8 tests as well?

Reply:
Makes sense!